scipy/cluster/tests/test_vq.py

```diff
@@ -4,25 +4,24 @@
 
 import numpy as np
 from numpy.testing import (
-    assert_array_equal, assert_allclose, assert_equal, assert_, suppress_warnings
+    assert_array_equal, assert_equal, assert_, suppress_warnings
 )
 import pytest
 from pytest import raises as assert_raises
 
 from scipy.cluster.vq import (kmeans, kmeans2, py_vq, vq, whiten,
                               ClusterError, _krandinit)
 from scipy.cluster import _vq
-from scipy.conftest import (
-    array_api_compatible,
-    skip_if_array_api,
-    skip_if_array_api_gpu
-)
+from scipy.conftest import array_api_compatible
 from scipy.sparse._sputils import matrix
 
 from scipy._lib._array_api import (
     SCIPY_ARRAY_API, copy, cov, xp_assert_close, xp_assert_equal
 )
 
+pytestmark = [array_api_compatible, pytest.mark.usefixtures("skip_if_array_api")]
```

Comment:
Indeed the `usefixtures` is awkward, but definitely worth it considering the number of improvements we have here. It seems like a workaround may be possible if we implement a custom pytest hook, although that may be more effort than it's worth.

I personally am a fan of this module-level opt in, mainly because of the concision, but also because it makes it very difficult to accidentally write tests which pass for NumPy but fail to have coverage of other backends. I think it's good to use this once a module is 'complete'.